src/scripts/hooks/use-markers.ts

```diff
@@ -27,12 +31,13 @@ export const useMarkers = (viewer: Viewer | null, markers: Marker[]) => {
       scene.canvas as HTMLCanvasElement
     );
 
-    handler.setInputAction(movement => {
-      const pickedObject = scene.pick(movement.position);
-      if (defined(pickedObject)) {
-        history.push(`/stories/${pickedObject.id._id}/0`);
-      }
-    }, ScreenSpaceEventType.LEFT_CLICK);
+    markerLink &&
```

Comment:
Thinking about it the `markerLink` property is not really a nice way to solve this. It would be cleaner and more flexible if we add a `link` property to each marker. You can add properties to cesium entities with https://cesium.com/docs/cesiumjs-ref-doc/Entity.html#properties
This way each marker can decide if it wants to be a link or not (by nor setting the link propery)